JS/form-validation.js

```diff
@@ -0,0 +1,8 @@
+document.getElementById('contact').addEventListener('submit', (event) => {
+  const email = document.getElementById('mail').value;
+  if (email !== email.toLowerCase()) {
+    event.preventDefault();
+    document.querySelector('.error').innerHTML = 'Your email address should be in lowercase';
+    event.target.reset();
+  }
+});
```

Comment:
- [ ] I am trying to put an email with Upper case letters and it is submitting as usual. This is not the expected behavior. It is supposed to submit only if the email is all in lowercase. Please correct the above code because I think they are supposed to work but I do not know what is wrong with it.